rotkehlchen/exchanges/coinbase.py

```diff
@@ -95,6 +95,57 @@ def trade_from_coinbase(raw_trade: Dict[str, Any]) -> Optional[Trade]:
     )
 
 
+def trade_from_conversion(trade_a: Dict[str, Any], trade_b: Dict[str, Any]) -> Optional[Trade]:
+    """Turn information from a conversion into a trade
+
+    Mary raise:
+    - UnknownAsset due to Asset instantiation
+    - DeserializationError due to unexpected format of dict entries
+    - KeyError due to dict entires missing an expected entry
+    """
+    # Check that the status is complete
+    if trade_a['status'] != 'completed':
+        return None
+
+    # Trade b will represent the asset we are converting to
+    if trade_b['amount']['amount'].startswith('-'):
+        trade_a, trade_b = trade_b, trade_a
+
+    timestamp = deserialize_timestamp_from_date(trade_a['updated_at'], 'iso8601', 'coinbase')
+    trade_type = deserialize_trade_type('sell')
+    tx_amount = deserialize_asset_amount(trade_b['amount']['amount'])
+    tx_asset = asset_from_coinbase(trade_b['amount']['currency'], time=timestamp)
+    native_amount = deserialize_asset_amount(trade_b['native_amount']['amount'])
+    native_asset = asset_from_coinbase(trade_b['native_amount']['currency'], time=timestamp)
+    # in coinbase you are buying/selling tx_asset for native_asset
+    pair = TradePair(f'{tx_asset.identifier}_{native_asset.identifier}')
```

Comment:
this won't work. The identifier is a rotki concept. Some times it may be the same as the symbol but this is not guaranteed. And especially with the changes we are making right now, all eth tokens will have a `_ceth_0xaddress` identifier. Here you need an exchange mapping.

Checking the code in my branch I think coinbase is the only exchange missing a mapping api endpoint. And we also don't have a `COINBASE_TO_WORLD` map.

So perhaps an approach to try here is to use `symbol`?

```suggestion
    pair = TradePair(f'{tx_asset.symbol}_{native_asset.symbol}')
```